shared/templates/selinux_booleans.csv

```diff
@@ -162,13 +162,13 @@ named_tcp_bind_http_port,
 named_write_master_zones,
 neutron_can_network,
 nfsd_anon_write,
-nfs_export_all_ro,
-nfs_export_all_rw,
+nfs_export_all_ro,enable
+nfs_export_all_rw,enable
 nis_enabled,
-nscd_use_shm,
+nscd_use_shm,enable
 openshift_use_nfs,
-openvpn_can_network_connect,
-openvpn_enable_homedirs,
+openvpn_can_network_connect,disable
```

Comment:
The base STIG should disable everything, akin to the default firewalld rules. If people need VPN, they'll have to open ports + enable booleans